main.js

```diff
@@ -0,0 +1,15 @@
+const source = $('#user-template').html()
+const template = Handlebars.compile(source)
+
+
+$.ajax({
+    method: "GET",
+    url: "https://randomuser.me/api/?inc=name,email&nat=gb&results=10",
+    success: (users) => {
+        for (user of users.results) {
```

Comment:
When declaring a new variable you should declare it with _const_ or _let_, otherwise, the variable will be defined as "_var_"